shell/platform/embedder/embedder.cc

```diff
@@ -51,14 +51,14 @@ extern const intptr_t kPlatformStrongDillSize;
   })()
 
 #define LOG_EMBEDDER_ERROR(code)                                      \
-  ({                                                                  \
+  ([=]() {                                                            \
     do {                                                              \
       FML_LOG(ERROR) << "Returning error '" << #code << "' (" << code \
                      << ") from Flutter Embedder API call to '"       \
                      << __FUNCTION__ << "'.";                         \
     } while (0);                                                      \
-    (code);                                                           \
-  })
+    return code;                                                      \
```

Comment:
imho it would be simpler to turn this into a function plus a helper macro.  something like:
```
static LogEmbedderError(int code, const char* name, const char* function) {
  FML_LOG(ERROR) << ...
  return code
}

#define LOG_EMBEDDER_ERROR(code) LogEmbedderError(code, #code, __FUNCTION__)
```

Reply:
I think this is better.  I'll change it to that